aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -25,8 +25,13 @@ void FunctionalTensorWrapper::set_constructor_metadata() {
   level_ = -1;
   // mirror all of the generic tensor metadata onto the wrapper
   copy_generic_tensor_metadata(value_.getIntrusivePtr().get(), this);
-  refresh_numel();
-  refresh_contiguous();
+  if (!has_symbolic_sizes_strides_) {
+    refresh_numel();
+    refresh_contiguous();
+  } else {
+    extra_meta_->numel_ = value_.sym_numel();
+    // TODO: contiguity
```

Comment:
@voznesenskym is going to refactor this logic so we consistently use the refresh methods to update numel and contiguous.

Reply:
this is up for grabs

Reply:
I'm working this